icd/api/pipeline_binary_cache.cpp

```diff
@@ -708,12 +708,12 @@ Util::Result PipelineBinaryCache::InjectBinariesFromDirectory(
         if (result == Util::Result::Success)
         {
             // Store each file into cache
-            strcpy(filePath, settings.devModeElfReplacementDirectory);
-            strcat(filePath, "\\");
+            Util::Strncat(filePath, sizeof(filePath), settings.devModeElfReplacementDirectory);
```

Comment:
Is there a reason you aren't using Strncpy?

Reply:
Good catch, thanks.